### ISSUE TITLE
ComboBox alerts when saturation changes

### ISSUE THREAD

**Issue by Matthew-phet** [phetsims/molarity#70]
@twant @zepumph During todays meeting and I discussed an issue with the combobox where sometimes when the solute is changed the solution goes from saturated to not saturated and vice versa. Is this something we want to include an alert for or add to the existing alert for the combobox.

**Comment by pixelzoom** [comment]
Not hugely important in GitHub issues.  But PhET has standardized on `ComboBox` and `comboBox`.  And if you were to use `combobox` in code or comments, it would fail lint.

**Comment by zepumph** [comment]
Thanks for the reminder! I always forget (thank golly g for that lint rule) and thus didn't see that edit while we were creating it.

**Comment by pixelzoom** [comment]
> (thank golly g for that lint rule)

Yes and no. I wouldn't be leaning on the lint rule so often if our convention was consistent.  `ComboBox`, `Checkbox`, and (now) `role='combobox'` makes it difficult to stay in the box (Box?)

**Comment by pixelzoom** [comment]
... AccordionBox, AlignBox, LayoutBox, HBox, VBox, BlackBox, WhiteBox,...  Let's just forget I even said anything :)

**Comment by terracoda** [comment]
@Matthew-phet great question. I'll need to mull it over, but my first thought is that you would want the alert to make it clear that a new solution is in the beaker and the state of the solution is likely important enough to alert.

But we can't describe everything all the time.
What's most important about changing the solute?
What are students exploring when they play will different solutes without playing with the sliders in between solute changes?
